### DISCUSSION TITLE
vant filed实现动态表单有相关的demo吗

### DISCUSSION THREAD

**Discussion by lovehebe330** [youzan/vant#9322] [Q&A]
### 设备 / 浏览器
移动

### Vant 版本
3.2.1

### Vue 版本
3.2.4

### 重现链接
<a href="https://vant-contrib.gitee.io/vant/v3/#/zh-CN/form" target="_blank">https://vant-contrib.gitee.io/vant/v3/#/zh-CN/form</a>

### 描述问题
van-field组件怎样实现动态表单？
![image](https://user-images.githubusercontent.com/20655248/130545373-f81f7466-844f-48c1-8025-14546473f330.png)

vant-file组件要实现动态添加删除表单，有具体的demo吗

**Comment by applelizihao** [comment]
这个很简单
